receiver/gitlabreceiver/internal/scraper/gitlabscraper/factory.go

```diff
@@ -42,8 +42,7 @@ func (f *Factory) CreateMetricsScraper(
 	conf := cfg.(*Config)
 	s := newGitLabScraper(ctx, params, conf)
 
-	return scraperhelper.NewScraper(
-		metadata.Type,
+	return scraperhelper.NewScraperWithoutType(
```

Comment:
Why are we not including the Type?

Reply:
In v0.114.0 of the collector NewScraper is deprecated and replaced with NewScraperWithoutType. The scraper type is now set in the configuration with the new function AddScraperWithType, replacing the deprecated AddScraper, which is shown [here](https://github.com/liatrio/liatrio-otel-collector/pull/599/files/cb533d8fe52c233525f766ea7207fc3db801c181#diff-202c4e844396fea996c376a22ed6baf6be601d1968b0ee49d813ef88b9b4a0e2R105)